MapboxAndroidDemo/build.gradle

```diff
@@ -68,6 +77,11 @@ android {
     lintOptions {
         abortOnError false
     }
+
+    play {
+        track = 'production'
```

Comment:
Publishing straight to `production` track rather than `alpha`?

Reply:
I'm trying to make the process as easy as possible so we can release a new version with every new maps release.